doc/platforms.md

```diff
@@ -12,15 +12,9 @@ Flame keeps it support on the stable and beta channel, dev and master channel sh
 
 To use Flame on web you need to be on the beta channel of Flutter and the game should be running using skia rendering, otherwise weird side effects may happen.
 
-To run your game using skia in release mode, use the following command:
+To run your game using skia, use the following command:
 
-`$ flutter run -d Chrome --release --dart-define=FLUTTER_WEB_USE_SKIA=true`
-
-To run with skia on debug mode, just start the project using the conventional command:
-
-`$ flutter run -d Chrome`
-
-And when the app starts and the debug is setup, press the `k` key. That will enable the `CanvasKit` mode.
+`$ flutter run -d Chrome --dart-define=FLUTTER_WEB_USE_SKIA=true`
```

Comment:
AWESOME! now we can run on non-release mode!

Reply:
YESSSSSS!!!! 🥳